### PR TITLE
Inject secrets into the same namespace as the binding

### PR DESCRIPTION
**Pull request by arschles**
Partially addresses https://github.com/kubernetes-incubator/service-catalog/issues/162
Requires #256

TODO: modify the code in uninject after https://github.com/kubernetes-incubator/service-catalog/pull/256 is merged

cc/ @pmorie

### CODE REVIEW

**Comment by duglin** [comment]
so far so good to me

File: pkg/controller/injector/k8s_binding_injector.go [hunk 1]
```diff
@@ -55,7 +55,8 @@ func CreateK8sBindingInjector() (BindingInjector, error) {
 func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *brokerapi.Credential) error {
 	secret := &v1.Secret{
 		ObjectMeta: v1.ObjectMeta{
-			Name: binding.Name,
+			Name:      binding.Name,
```
Comment (pmorie) [inline pkg/controller/injector/k8s_binding_injector.go]: This is fine for now, but should be the name of the specified secret if that is set.

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
given we're short on approvers today this is trivial I'm gonna merge.

**Pull request merged**